requirements.txt

```diff
@@ -2,7 +2,7 @@ bz2file==0.98.*
 coloredlogs==10.0.*
 cycler==0.10.*
 Cython==0.29.*
-dipy==1.1.*
+dipy>=1.1.*
```

Comment:
dipy>=1.1.0 ? What the behavior of a * and a >= together

Reply:
I will do dipy>=1.1